src/components/tooltip/tooltip.js

```diff
@@ -3,391 +3,428 @@
  * @name material.components.tooltip
  */
 angular
-  .module('material.components.tooltip', [ 'material.core' ])
-  .directive('mdTooltip', MdTooltipDirective)
-  .service('$$mdTooltipRegistry', MdTooltipRegistry);
+    .module('material.components.tooltip', ['material.core'])
```

Comment:
As mentioned in another comment, this should depend on the panel module as well.